### PR TITLE
fix(safe-apps): Remove trailing slash from permissions appUrl comparisons

### PR DESCRIPTION
**Pull request by yagopv**
## What it solves
Resolves https://github.com/safe-global/safe-react-apps/issues/500

## How this PR fixes it
We removed the trailingSlash from the comparisons between origins

### CODE REVIEW

File: src/hooks/safe-apps/permissions/useSafePermissions.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { Permission, PermissionCaveat, PermissionRequest } from '@gnosis.pm/safe
 
 import { PermissionStatus } from '@/components/safe-apps/types'
 import useLocalStorage from '@/services/local-storage/useLocalStorage'
+import { trimTrailingSlash } from '../../../utils/url'
```
Comment (mikhailxyz) [inline src/hooks/safe-apps/permissions/useSafePermissions.ts:7]: nit: could you use the `@` alias?

File: src/hooks/safe-apps/permissions/useBrowserPermissions.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { AllowedFeatures, PermissionStatus } from '@/components/safe-apps/types'
 import useLocalStorage from '@/services/local-storage/useLocalStorage'
 import { useCallback } from 'react'
+import { trimTrailingSlash } from '../../../utils/url'
```
Comment (mikhailxyz) [inline src/hooks/safe-apps/permissions/useBrowserPermissions.ts:4]: nit: could you use the `@` alias?

**Review by mikhailxyz — APPROVED** [review]

File: src/hooks/safe-apps/permissions/useSafePermissions.ts (hunk 1 above)
Comment (yagopv) [reply src/hooks/safe-apps/permissions/useSafePermissions.ts]: done

File: src/hooks/safe-apps/permissions/useBrowserPermissions.ts (hunk 2 above)
Comment (yagopv) [reply src/hooks/safe-apps/permissions/useBrowserPermissions.ts]: done

**Pull request merged**